src/utils.jl

```diff
@@ -1,11 +1,10 @@
 function linshrink(F::Union{UniformScaling, AbstractMatrix},
                    S::AbstractMatrix, λ::Real)
-    return Symmetric((1.0 - λ)*S + λ*F)
+    return Symmetric((1.0 .- λ).*S .+ λ.*F)
```

Comment:
Broadcasting won't work when `F` is `UniformScaling`, unfortunately. I think this method could be split into two, one for `UniformScaling` and a second one for `AbstractMatrix`.